### PR TITLE
Add `strings` label when res/values dir changes

### PR DESCRIPTION
**Pull request by krmanik**
## Purpose / Description
Automate process of adding `strings` label when values dir changed.

## Fixes
Fixes _Link to the issues._

## Approach
1. Run git diff on res/values dir 
2. Use the result to run the github actions script

## How Has This Been Tested?
Tested on this PR
https://github.com/krmanik/Anki-Android-Copy/pull/1

## Learning (optional, can help others)
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison** [comment]
I'd accept this as-is, regardless of the below

- Could a note to the maintainers be added to explain the pre-merge and post-merge actions which should be taken
- Could this use the same definitions as the "CrowdIn sync" scripts, to ensure we're not adding this label to constants files which would not be synced

**Comment by krmanik** [comment]
I think scripts need more improvement, I am testing here so I marked it as draft.
https://github.com/krmanik/Anki-Android-Copy/pull/2/files

**Comment by krmanik** [comment]
> I'd accept this as-is, regardless of the below. It's a step in the right direction. If these aren't accepted for the PR, please let me know
> 
> * Could a note to the maintainers be added to explain the pre-merge and post-merge actions which should be taken
> * Could this use the same definitions as the "CrowdIn sync" scripts, to ensure we're not adding this label to constants files which would not be synced

I will update with the suggestions.

**Comment by krmanik** [comment]
Test result 
https://github.com/krmanik/Anki-Android-Copy/pull/2

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -51,4 +52,78 @@ jobs:
...
+                labels: labels
+              });
+            }
+
+            async function addComments() {
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: `Message to maintainers, this PR contains strings changes. So read more about updating strings on wiki,
```
Comment (krmanik) [inline .github/workflows/label.yml:101]: Needs better message here.

File: .github/workflows/label.yml [hunk 2]
```diff
@@ -51,4 +52,78 @@ jobs:
...
+                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#localization-administration
+                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#download-localized-strings`
```
Comment (mikehardy) [inline .github/workflows/label.yml:103]:
```suggestion
                body: `Message to maintainers, this PR contains strings changes. 
                1. Before merging this PR, it is best to run the "Sync Translations" github action,
                    then make and merge a PR from the i18n_sync branch to get translations cleaned out.
                2. Then merge this PR, and immediately do another translations PR so the huge change made
                     by this PR's key changes are all by themselves.
                
                Read more about updating strings on wiki,
                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#localization-administration
                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#download-localized-strings`
```

What do you think? Are double quotes allowed in the string? Probably but worth asking.

**Review by mikehardy — APPROVED** [review]
Results look great on your fork. Consider this an approve, but I did make a suggestion on the wording. That's just a style preference though, take my wording, come up with your own, whatever you think is right then please feel free to merge this! Has very low risk of harming anything, no problem getting it in and seeing how it works in practice.

Thanks for this Mani!

File: .github/workflows/label.yml (hunk 2 above)
Comment (krmanik) [reply .github/workflows/label.yml]: This message is better. Double quotes are allowed.
Thanks

**Review by mikehardy — APPROVED** [review]
LGTM!

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
This is great, thank you!

**Pull request merged**